jobs/forecast.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/https'
+require 'json'
+
+# Forecast API Key from https://developer.forecast.io
+$forecast_api_key = "9f9ea934981bbeae40889866c7cc2fe5"
```

Comment:
Please remove api key and make it an environment variable

Reply:
Done